runtime/hydradx/src/assets.rs

```diff
@@ -431,6 +437,79 @@ impl pallet_dca::Config for Runtime {
 	type WeightInfo = weights::dca::HydraWeight<Runtime>;
 }
 
+// Provides weight info for the router
+pub struct AmmWeights;
+impl AmmWeights {
+	pub fn sell_overhead_weight() -> Weight {
+		weights::route_executor::HydraWeight::<Runtime>::sell_in_lbp()
```

Comment:
it would be probably worth to comment what this does :) i can imagine looking at this in 6mounths being like .... what?